docs/guides/modules/orchestrate/pages/set-up-triggers.adoc

```diff
@@ -0,0 +1,121 @@
+= Set up triggers
+:page-platform: Cloud, Server v4+
+:page-description: A guide to setting up triggers for your pipelines.
+:experimental:
+
+Triggers are configurable rules that determine when to automatically run a pipeline based on specific events and conditions.
+
+A project can have one or more triggers, which are associated with events from a source of change. The most common type of trigger is a VCS event, for example, a push to a branch on a GitHub repository. Any system that can emit outbound webhooks can be set up as a trigger. A trigger can have only one pipeline but a pipeline can have multiple triggers.
+
+A trigger kicks off a pipeline to run the workflows defined in your connected config source.
+
+== Add a trigger
+
+NOTE: Bitbucket Cloud triggers are created automatically when setting up a project and they cannot be edited. The only additional trigger types available for Bitbucket Cloud pipelines are xref:scheduled-pipelines.adoc[scheduled triggers].
```

Comment:
I like that here you refer to them as **schedule triggers**, in other places they're referred to as scheduled _pipelines_. 
I will open a PR soon to start working on changing that language